gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -724,6 +906,169 @@ def call_metric(self, other_args: List[str]):
                 self.stocks_data,
             )
 
+    def call_bs(self, other_args: List[str]):
```

Comment:
Let's merge these 3 into one. Also, here you are falling into the same duplication issue. This is bad because let's say there was a new metric, you would have to update it in 3 places which increases a lot the chances of future bugs.

Instead, you can iterate through your dictionary and do something like:
```
for k, v in sa_keys["BS"].items():
   help_text += f"  {k} {(20 - len(k)) * ' '} {v}"
```